applications/neuroscan/frontend/src/components/viewers/viewer.js

```diff
@@ -145,18 +145,24 @@ class Viewer extends React.Component {
     }
   }
 
-  handleAddInstancesToViewer = async (viewerId = null) => {
-    const { addInstancesToViewer, timePoint } = this.props;
+  handleCloneViewerWithInstancesList = async (addToViewerId = null) => {
```

Comment:
I understand the change in the name of this method but given the fact that there's a path where the clone doesn't happen I'm not sure if the renaming was progress.

Reply:
agree @afonsobspinto I will revert the rename